tests/components/ipma/test_config_flow.py

```diff
@@ -1,116 +1,57 @@
 """Tests for IPMA config flow."""
-from unittest.mock import Mock, patch
-
-from homeassistant.components.ipma import config_flow
-from homeassistant.const import CONF_LATITUDE, CONF_LONGITUDE
-from homeassistant.core import HomeAssistant
+from unittest.mock import patch
 
+import pytest
 
-async def test_show_config_form() -> None:
-    """Test show configuration form."""
-    hass = Mock()
-    flow = config_flow.IpmaFlowHandler()
-    flow.hass = hass
+from homeassistant import config_entries
+from homeassistant.components.ipma.const import DOMAIN
+from homeassistant.const import CONF_LATITUDE, CONF_LONGITUDE, CONF_NAME
+from homeassistant.core import HomeAssistant
 
-    result = await flow._show_config_form()
 
-    assert result["type"] == "form"
-    assert result["step_id"] == "user"
+@pytest.fixture(name="ipma_setup", autouse=True)
+def ipma_setup_fixture(request):
+    """Patch ipma setup entry."""
+    if "disable_autouse_fixture" in request.keywords:
+        yield
+    else:
+        with patch(
+            "homeassistant.components.ipma.async_setup_entry", return_value=True
+        ):
+            yield
 
 
-async def test_show_config_form_default_values() -> None:
+async def test_show_config_form(hass: HomeAssistant) -> None:
```

Comment:
Please rename this test + update it to not only show the user step form, but actually finish the user flow, i.e. add steps doing
```py
result = await hass.config_entries.flow.async_configure(
    result["flow_id"],
    ....
)
assert result == ...
```